+srv/expServer.m

```diff
@@ -345,6 +360,29 @@ function calibrateWaterDelivery()
     save(rigHwFile, 'daqController', '-append');
   end
 
+    function viewCalibration()
+        
+        calib = rig.daqController.SignalGenerators(rewardId).Calibrations(end);
+        
+        fig = figure; hold on
+        plot([calib.measuredDeliveries.durationSecs],...
+            [calib.measuredDeliveries.volumeMicroLitres],'x-')
+        if isfield(calib, 'dateTime')
```

Comment:
Is there a case where `dateTime` isn't a field? It's assumed to be there in other parts of the code...

Reply:
It was coming up for me, but I think it's an issue that's specific to some things I had done on my branch that re fixed now. I think in dev it's safe to assume dateTime will always be there if a calibration in.

Reply:
Okay, I removed the if statement because on master there would be an error thrown beforehand, here: https://github.com/cortex-lab/Rigbox/blob/7d6b6010d9166632ba5a7181ed9414ea866480f7/%2Bhw/devices.m#L54-L57